docs/docs/intro/configure.md

```diff
@@ -375,24 +375,121 @@ marathon {
 </TabItem>
 </Tabs>
 
-### Strict mode
-By default, if one of the test retries succeeds then the test is considered successfully executed. If you require success status only when
-all retries were executed successfully you can enable the strict mode. This may be useful to verify that flakiness of tests was fixed for
-example.
+### Execution strategy
+When executing tests with retries there are multiple trade-offs to be made. Two execution strategies are supported: any success or all success.
+By default, any success strategy is used with fast execution i.e. if one of the test retries succeeds then the test is considered successfully
```

Comment:
Worth changing all instances of "any success" & "all success" to `ANY_SUCCESS` & `ALL_SUCCESS` for easier reading?